account_reconcile_compassion/models/res_config.py

```diff
@@ -19,28 +18,28 @@ class AccountConfigSettings(models.TransientModel):
     currency exchange.
     """
 
-    _inherit = 'account.config.settings'
+    _inherit = "account.config.settings"
 
     currency_exchange_analytic_account = fields.Many2one(
-        'account.analytic.account',
-        default=lambda s: s.get_currency_exchange_analytic_account()
+        "account.analytic.account",
+        default=lambda s: s.get_currency_exchange_analytic_account(),
     )
 
     @api.multi
     def set_currency_exchange_analytic_account(self):
```

Comment:
The `set_currency...` and `get_currency...` methods can be deleted.
Calls to `get_currency_` will then need refactoring